src/tree/node_children.rs

```diff
@@ -559,31 +559,31 @@ mod inner {
 
         /// Access to the nodes array.
         #[inline(always)]
-        pub fn nodes<'a>(&'a self) -> &'a [Arc<Node>] {
+        pub fn nodes(&self) -> &[Arc<Node>] {
```

Comment:
It's a bit too verbose for me, but I pushed a version with the `'_` placeholder lifetime that might be a reasonable middle ground?

Reply:
Hmm.  You know, just go with your original change.  I like the verbosity in this case, but if you're reaching for compromises that probably means you feel more strongly about it than I do, ha ha. :-)

Reply:
Or put another way: using `'_` kind of seems like the worst of both worlds: it's simultaneously implicit (which I don't like) *and* verbose (which you don't like).  And I'd prefer at least one of us be happy, rather than both unhappy, ha ha. :-)